### DISCUSSION TITLE
Question on how downloads are handled

### DISCUSSION THREAD

**Discussion by 3otgjrq** [UltimaHoarder/UltimaScraper#1125] [Q&A] [answered]
With all the fancy file downloading methodologies that were added over the last year, I'm curious how downloads are handled.
Lets say I run and scrape 25GB of data.
Then I move that data somewhere to be stored. Is there a meta file that contains the file names etc I can keep in the .sites directory that would prevent re-downloading? If something like that isn't part of the workflow yet, is there any chance of implementing it?

**Comment by UltimaHoarder** [comment, marked as answer]
All download history is located in metadata files. If you download something successfully, it won't be downloaded again if you move it else where, BUT if you delete the metadata file, it will download again.
